circle.yml

```diff
@@ -21,15 +21,24 @@ machine:
     # Manually install very recent version of cassandra; unless we create a user we have to run it as root
     - curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
     - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
+    ## Apply system resource limits to Cassandra
+    ## As seen at https://github.com/circleci/image-builder/blob/8d1bb13b614da27447da0cd70d292e867395316b/circleci-provision-scripts/misc.sh#L63-L65
+    - sed -i 's/system_memory_in_mb=.*/system_memory_in_mb=2048/' apache-cassandra-*/conf/cassandra-env.sh
+    - sed -i 's/system_cpu_cores=.*/system_cpu_cores=1/' apache-cassandra-*/conf/cassandra-env.sh
+    - sed -i 's/JVM_OPTS="$JVM_OPTS -Xms${MAX_HEAP_SIZE}"/JVM_OPTS="$JVM_OPTS -Xms256M"/' apache-cassandra-*/conf/cassandra-env.sh
```

Comment:
note that jvm options are now found in `conf/jvm.options`

but this will still work fine.